prow/testutil/fixtures.go

```diff
@@ -29,19 +29,19 @@ import (
 
 // CompareWithFixtureDir will compare all files in a directory with a corresponding test fixture directory.
 func CompareWithFixtureDir(t *testing.T, golden, output string) {
-	if walkErr := filepath.Walk(golden, func(path string, info os.FileInfo, err error) error {
+	if walkErr := filepath.Walk(output, func(path string, info os.FileInfo, err error) error {
 		if err != nil {
 			return err
 		}
-		if info.IsDir() {
+		if info.IsDir() || info.Mode()&os.ModeSymlink == os.ModeSymlink {
```

Comment:
```
 --- FAIL: TestCensorIntegration (0.01s)
    fixtures.go:68: failed to read testdata file: open testdata/output/artifacts/first/link: no such file or directory 
```

I explicitly do *not* look at symlinks in this test, so seeing this failure makes me think that somehow this isn't a symlink in bazel-land. Sigh.